src/pages/search/Search.tsx

```diff
@@ -0,0 +1,164 @@
+    lat: 37,
+    lng: -117,
+    listingPicUrl: 'https://apod.nasa.gov/apod/image/1902/MagneticOrion_EsoSofia_1833.jpg',
+    pricePerNightUsd: 42,
+    prices: [],
+    sleepingArrangement: '2 Beds',
+    state: 'CA',
+    title: 'Actual nest of bees'
+  },
+  {
+    city: 'San Francisco',
+    country: 'USA',
+    id: 'foo',
+    idSlug: 'fooooo',
+    homeType: 'Hotel',
+    lat: 37,
+    lng: -117,
+    listingPicUrl: 'https://apod.nasa.gov/apod/image/1902/MagneticOrion_EsoSofia_1833.jpg',
+    pricePerNightUsd: 42,
+    prices: [],
+    sleepingArrangement: '2 Beds',
+    state: 'CA',
+    title: 'Actual nest of actual terrifying bees'
+  },
+  {
+    city: 'San Francisco',
+    country: 'USA',
+    id: 'foo',
+    idSlug: 'fooooo',
+    homeType: 'Hotel',
+    lat: 36,
+    lng: -116,
+    listingPicUrl: 'https://apod.nasa.gov/apod/image/1902/MagneticOrion_EsoSofia_1833.jpg',
+    pricePerNightUsd: 42,
+    prices: [],
+    sleepingArrangement: '2 Beds',
+    state: 'CA',
+    title: 'Actual nest of bees'
+  },
+  {
+    city: 'San Francisco',
+    country: 'USA',
+    id: 'foo',
+    idSlug: 'fooooo',
+    homeType: 'Hotel',
+    lat: 37,
+    lng: -117,
+    listingPicUrl: 'https://apod.nasa.gov/apod/image/1902/MagneticOrion_EsoSofia_1833.jpg',
+    pricePerNightUsd: 42,
+    prices: [],
+    sleepingArrangement: '2 Beds',
+    state: 'CA',
+    title: 'Actual nest of bees'
+  },
+  {
+    city: 'San Francisco',
+    country: 'USA',
+    id: 'foo',
+    idSlug: 'fooooo',
+    homeType: 'Hotel',
+    lat: 37,
+    lng: -117,
+    listingPicUrl: 'https://apod.nasa.gov/apod/image/1902/MagneticOrion_EsoSofia_1833.jpg',
+    pricePerNightUsd: 42,
+    prices: [],
+    sleepingArrangement: '2 Beds',
+    state: 'CA',
+    title: 'Actual nest of bees'
+  },
+  {
+    city: 'San Francisco',
+    country: 'USA',
+    id: 'foo',
+    idSlug: 'fooooo',
+    homeType: 'Hotel',
+    lat: 37,
+    lng: -117,
+    listingPicUrl: 'https://apod.nasa.gov/apod/image/1902/MagneticOrion_EsoSofia_1833.jpg',
+    pricePerNightUsd: 42,
+    prices: [],
+    sleepingArrangement: '2 Beds',
+    state: 'CA',
+    title: 'Actual nest of bees'
+  },
+  {
+    city: 'San Francisco',
+    country: 'USA',
+    id: 'foo',
+    idSlug: 'fooooo',
+    homeType: 'Hotel',
+    lat: 37,
+    lng: -117,
+    listingPicUrl: 'https://apod.nasa.gov/apod/image/1902/MagneticOrion_EsoSofia_1833.jpg',
+    pricePerNightUsd: 42,
+    prices: [],
+    sleepingArrangement: '2 Beds',
+    state: 'CA',
+    title: 'Actual nest of bees'
+  },
+  {
+    city: 'San Francisco',
+    country: 'USA',
+    id: 'foo',
+    idSlug: 'fooooo',
+    homeType: 'Hotel',
+    lat: 37,
+    lng: -117,
+    listingPicUrl: 'https://apod.nasa.gov/apod/image/1902/MagneticOrion_EsoSofia_1833.jpg',
+    pricePerNightUsd: 42,
+    prices: [],
+    sleepingArrangement: '2 Beds',
+    state: 'CA',
+    title: 'Actual nest of beeeeeeeees'
+  }
+];
+
+const Search = () => {
+  return (
+    <>
```

Comment:
i think for a lot of our fragments we can use the `<Fade>` component instead now to have fade-in page transitions